src/App.vue

```diff
@@ -0,0 +1,294 @@
+<template>
+  <div>
+    <div class="focus">
+      <input type="text" ref="inputRef" />
+      <br /><br />
+      <div class="button-container">
+        <button @click="inputFieldFocus">Focus</button>
+        <button @click="toggleModal">Open Modal</button>
+      </div>
+
+    </div>
+    <Modal :isVisible='showModal' :title='modalTitle' :content='modalContent' :theme='modalTheme'
```

Comment:
improve component naming convension https://vuejs.org/style-guide/rules-strongly-recommended.html @Shashank-002 
@piyush read it too